check-diff.sh

```diff
@@ -565,15 +564,16 @@ function run_phpunit_local {
 	(
 		echo "## phpunit"
 		if [ -n "$( type -t phpunit )" ] && [ -n "$WP_TESTS_DIR" ]; then
-			if [ -n "$PHPUNIT_CONFIG" ] || [ -e phpunit.xml* ]; then
+			if [ -n "$PHPUNIT_CONFIG" ]; then
```

Comment:
@westonruter @valendesigns I've pushed another fix to stop tests running twice.

I could have misunderstood something in the broader context of WP-Dev-Lib, but what I'm doing is if a PHPUNIT_CONFIG is set, run only that config file, otherwise loop through the project and find the phpunit.xml files and run them one by one.

Thoughts?